src/components/ui-shell/header-menu-button.ts

```diff
@@ -94,8 +96,8 @@ class BXHeaderMenuButton extends FocusMixin(LitElement) {
       [`${prefix}--header__action--active`]: active,
     });
     return html`
-      <button class="${classes}" ?disabled=${disabled} aria-label="${ifNonNull(buttonLabel)}" @click=${handleClick}>
-        ${active ? Close20() : Menu20()}
+      <button part="button" class="${classes}" ?disabled=${disabled} aria-label="${ifNonNull(buttonLabel)}" @click=${handleClick}>
+        ${(active ? Close20 : Menu20)({ slot: 'toggle-icon' })}
```

Comment:
@asudoh Quick question, should this not be `16`, based on [this comment](https://github.com/carbon-design-system/carbon-web-components/pull/502#issuecomment-727121316)? Just trying to understand better what the base is for font/icon/spacing sizes. Since this one is `20`, I'm just trying to understand better 🙏